bin/createteams.py

```diff
@@ -0,0 +1,150 @@
+# NOTE: click is not a dependency of this project, you'll have to manually
+#       install it to run this file.
+import click
```

Comment:
Rather than commenting, wrap this in a try/catch for ImportError that `print`s this message (and then exits)